lib/player.rb

```diff
@@ -0,0 +1,10 @@
+class Player
+  attr_accessor :name, :next_move, :moves, :token
+
+  def initialize(name, token)
+    @name = name
+    @next_move = []
+    @moves = []
+    @token = token
```

Comment:
Do you really need to store the player's moves?
What happens if you just mark the moves on the board and check for the player's mark in the winning positions?
That way you can simplify your `winner?` and `mark_board` methods.